src/engines/parachain_engine.rs

```diff
@@ -30,15 +34,15 @@ pub fn instantiate_template_dir(template: &Template, target: &Path, config: Conf
 			return instantiate_base_template(target, config);
 		},
 	};
-	clone_and_degit(url, target)?;
+	let tag = clone_and_degit(url, target)?;
 	Repository::init(target)?;
-	Ok(())
+	Ok(tag)
 }
 
-pub fn instantiate_base_template(target: &Path, config: Config) -> Result<()> {
+pub fn instantiate_base_template(target: &Path, config: Config) -> Result<String> {
 	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
 	let source = temp_dir.path();
-	clone_and_degit("https://github.com/r0gue-io/base-parachain", source)?;
+	let tag = clone_and_degit("https://github.com/r0gue-io/base-parachain", source)?;
```

Comment:
Same reason as down below. `tag` needs to be an Option i.e. `clone_and_degit` returns a `Result<Option<String>>`